satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -82,255 +82,316 @@ datasets:
+    standard_name: product_quality
 
   product_completeness_ctth:
     name: product_completeness_ctth
     file_type: nc_fci_ctth
     file_key: product_completeness
-    long_name: product_completeness_index
+    standard_name: product_completeness
 
   product_timeliness_ctth:
     name: product_timeliness_ctth
     file_type: nc_fci_ctth
     file_key: product_timeliness
-    long_name: product_timeliness_index
+    standard_name: product_timeliness
 
   # OCA
   retrieved_cloud_phase:
     name: retrieved_cloud_phase
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_phase
-    standard_name: thermodynamic_phase_of_cloud_water_particles_at_cloud_top
+    standard_name: thermodynamic_phase_of_cloud_particles_classification
+    fill_value: -127
+    import_enum_information: True
 
   retrieved_cloud_optical_thickness:
     name: retrieved_cloud_optical_thickness
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_optical_thickness
-    long_name: cloud_optical_depth
+    standard_name: atmosphere_optical_thickness_due_to_cloud
 
   retrieved_cloud_optical_thickness_upper_layer:
     name: retrieved_cloud_optical_thickness_upper_layer
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_optical_thickness
     layer: 0
-    long_name: cloud_optical_depth
+    standard_name: atmosphere_optical_thickness_due_to_cloud
+
+  retrieval_error_cloud_optical_thickness_upper_layer:
+    name: retrieval_error_cloud_optical_thickness_upper_layer
+    resolution: 2000
+    file_type: nc_fci_oca
+    file_key: retrieval_error_cloud_optical_thickness
+    layer: 0
+    standard_name: atmosphere_optical_thickness_due_to_cloud standard_error
 
   retrieved_cloud_optical_thickness_lower_layer:
     name: retrieved_cloud_optical_thickness_lower_layer
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_optical_thickness
     layer: 1
-    long_name: cloud_optical_depth
+    standard_name: atmosphere_optical_thickness_due_to_cloud
+
+  retrieval_error_cloud_optical_thickness_lower_layer:
+    name: retrieval_error_cloud_optical_thickness_lower_layer
+    resolution: 2000
+    file_type: nc_fci_oca
+    file_key: retrieval_error_cloud_optical_thickness
+    layer: 1
+    standard_name: atmosphere_optical_thickness_due_to_cloud standard_error
 
   retrieved_cloud_particle_effective_radius:
-    name: retrieved_cloud_particle_effective_radius
+    name: retrieved_cloud_particle_effective_radius_upper_layer
     resolution: 2000
     file_type: nc_fci_oca
     file_key: retrieved_cloud_particle_effective_radius
-    standard_name: effective_radius_of_cloud_condensed_water_particles_at_cloud_top
+    standard_name: effective_radius_of_cloud_particles
 
-  retrieved_cloud_top_temperature:
-    name: retrieved_cloud_top_temperature
+  retrieval_error_cloud_particle_effective_radius:
+    name: retrieval_error_cloud_particle_effective_radius_upper_layer
```

Comment:
remove "_upper_layer" since this a 2D dataset, which is independent of the retrieved layers

Reply:
Fixed